docs/rfcs/1013-parsec-auth-scheme.md

```diff
@@ -0,0 +1,267 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+# Schema for parsec-auth
+
+## Overview
+
+This RFC defines the schema used for communicating with the parsec-auth service.
+
+## Goals and Non-Goals
+
+The goal is to define the required schema used when communicating with the parsec-auth service.
+For simplicity, we will skip the fido2 part that will be defined in a later RFC.
+
+## Design
+
+### Account creation
+
+To start the account creation, the client start by sending its email.
+
+```yml
+{
+  "cmd": "account_creation_step1_email_validation",
```

Comment:
Most of existing commands use the `<object>_<verb>` syntax  (`realm_create`, `vlob_create`, `user_revoke`, etc.) so I would suggest using `account_create`. Also, including a "step number" was used in the invite protocol and later removed.

So what about something like:
```suggestion
  "cmd": "account_create_get_token",
```
```suggestion
  "cmd": "account_create_get_validation_token",
```
```suggestion
  "cmd": "account_create_get_email_validation_token",
```

Reply:
Or
```suggestion
  "cmd": "account_send_email_validation_token",
```